sandbox/readaligner_pairhmm_train.py

```diff
@@ -28,6 +27,11 @@ def trusted_str(cov, trusted_cutoff):
 
 
 def main():
+    try:
```

Comment:
why try/pass here? just import it. tests should still succeed as we don't actually run this script in the tests.